### ISSUE TITLE
framestyles not working propertly in 3

### ISSUE THREAD

**Issue by Kelvyn88** [JuliaPlots/Plots.jl#1930]

**Comment by daschw** [comment]
Yes, framestyles are currently only implemented for 2D axes. However, it would be really nice to have this for 3D plots as well. I'm not sure if the different backends support it.
The gray background should be quite easy to fix. I can take a quick look.

**Comment by lovasoa** [comment]
Hello ! Any news on this ? I found myself blocked by this issue. Is there any workaround to disable the frame completely in 3D plots ?

Example of bogus framestyle : 

```julia
plot([1,2], [1,2], [0,0], framestyle=:none)
```

![image](https://user-images.githubusercontent.com/552629/57583438-f1c8d080-74d0-11e9-9ce9-38539ef2220c.png)

**Comment by daschw** [comment]
`plot([1,2], [1,2], [0,0], axiscolor = invisible())`. However this also hides the ticks ...

**Comment by lovasoa** [comment]
`axiscolor=invisible()`doesn't seem to work...

<img width="607" alt="image" src="https://user-images.githubusercontent.com/552629/57610249-b4b41b00-7570-11e9-9f62-abc9c841df9a.png">
